Message_Spam_Detection/Dataset_Cleaning.py

```diff
@@ -0,0 +1,77 @@
+#importing required libraries
+import pandas as pd
+import string 
+import nltk
+from sklearn.feature_extraction.text import CountVectorizer
+from sklearn.feature_extraction.text import TfidfTransformer
+from sklearn.ensemble import RandomForestClassifier
+import pickle
+import warnings
+import re
+warnings.filterwarnings("ignore")
+nltk.download('stopwords')
+nltk.download('wordnet')
+
+#reading the dataset 
+#dataset: https://www.kaggle.com/uciml/sms-spam-collection-dataset
+msg=pd.read_csv("dataset.csv",encoding='latin-1')
```

Comment:
There is no file named dataset.csv. Also, correct the path of the files would  be "./Message_Spam_Detection/filename"

Reply:
Okay. Will make the changes!